src/eth/mining.json

```
@@ -50,32 +50,32 @@
 	},
 	{
 		"name": "eth_submitWork",
-		"summary": "Used for submitting a proof-of-work solution.",
+		"summary": "Can be used by an external miner to submit their proof-of-work solution.",
 		"params": [
 			{
-				"name": "Proof-of-work hash",
+				"name": "nonce",
 				"required": true,
 				"schema": {
-					"$ref": "#/components/schemas/bytes32"
+					"$ref": "#/components/schemas/bytes"
```

```suggestion
					"$ref": "#/components/schemas/bytes8"
```
Can you add a `bytes8` type to `base-types.json` please? Check out how `bytes32` is implemented. While you're there, could be a good time to also fix #106.

---

Done